cmdriver/cmdriver.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    prometheus.io/port: "9101"
+    prometheus.io/scrape: "true"
+  labels:
+    app: cmdriver
+  name: cmdriver
+  namespace: scaletest
+spec:
+  template:
+    metadata:
+      labels:
+        app: cmdriver
+    spec:
+      containers:
+      - name: cmdriver
+        image: marceloamaral/cmdriver:latest
+        command:
+          - "/cmdriver"
+          - "-lambda=$(LAMBDA)"
+          - "-maxpop=$(MAXPOP)"
+          - "-n=$(NUM)"
+          - "-log_dir=$(LOGDIR)"
+          - "-waitBeforeTerminate=$(WAITBEFORETERMINATE)"
+        env:
+        - name: LAMBDA
+          value: "10"
```

Comment:
How does one use this with different values for these parameters?  Perhaps the Deployment was more convenient in this regard.

If we keep a single-run Job then it would perhaps be better to specify each environment variable on one line, like this:
```
    - {name: "LAMBDA", value: "10"}
```
That way there is some hope of automating edits to this.
```